packages/authorization-process/src/services/validators.ts

```diff
@@ -73,6 +74,20 @@ export const assertOrganizationIsPurposeConsumer = (
   }
 };
 
+export const assertOrganizationIsDelegate = (
```

Comment:
Fors consistency with agrreement & purpose services
```suggestion
export const assertRequesterIsDelegateConsumer = (
```